Using-Mastodon/FAQ.md

```diff
@@ -64,50 +71,114 @@ Try [TootyFruity](https://play.google.com/store/apps/details?id=ch.kevinegli.too
+
 #### What’s with the pineapples?
 They’re tasty, and they make people smile. Just go with it.
 
+#### I see “Awoo” a lot, what does that mean?
+Try saying it out loud. It’s fun!
+
 #### How do I delete my account?
 At the moment, you will have to toot the admin of your instance for help on that one.
 
+<a name="who-is-my-admin-and-how-do-i-contact-follow-them"></a>
 #### Who is my admin, and how do I contact/follow them?
 Click on **Getting Started**, then **Extended Information**. An info page displays. Hopefully, they put their contact information in there!
 
-![Getting Started](https://octodon.social/media/TKlZpSNRRcHBOEFNlGs) ![Extended Information](https://octodon.social/media/D7bfl1NYzNy4XIT8nLs) ![Info Page](https://octodon.social/media/zvoI8yUt4uurZcKqFRk)
+<img src="screenshots/toolbar-getting_started.png" alt="Getting Started Button" height="200"/>
+<img src="screenshots/getting_started-extended_information.png" alt="Extended Information" height="200"/>
+<img src="screenshots/admin_info.png" alt="Admin Info" height="200"/>
 
 #### Wow, I keep seeing offensive stuff from a particular instance, is there a way to block *all* of it?
 Not without a lot of clicking, sorry. But, your instance admin can do it! Please contact her.
 
+#### I blocked someone, why can I still see their posts?
+Blocking doesn’t catch when toots are boosted.
+
+#### Is Mastodon moderated?
+Each instance handles moderation differently, and each has their own moderators. It’s okay to ask what the rules are for the instance you are on. Usually the rules are posted on the instance's about page ([like the “more” page at Mastodon.social](https://mastodon.social/about/more))
+
+#### How do I report offensive content?
+Beneath each post, you will see three dots. Clicking on those will allow you to expand a post or report the content. When reporting content, select all posts that need moderator attention.
+
+#### How do I handle harassment?
+If your admin takes harassment seriously, you can report it to them through the post-reporting system. This can be found through the ellipsis (“…”) beneath the post, or by contacting your admin directly.
+
+#### Will my Private Messages reach people on other instances?
+Yes, they should. However, private messages are *not secure*. You should avoid sending sensitive information through them.
+
+#### What does “Adjust Status Privacy” mean and how does it work?
+The **globe** icon under the toot area adjusts your status privacy by changing who can see your posts. This is what happens:
+
+| Privacy setting | Destination | Notes |
+| --------------- | ----------- | ----- |
+| Public          | global, public timeline | Federated. People on other instances will probably see it. |
+| Unlisted        | local public timeline | Can be boosted and thus seen on other instances.  |
```

Comment:
This confused me, because there are 2 questions 1) who is the toot broadcasted to(federated, local, followers, etc.)? and 2) who can *access* the toot? So I changed "Destination" to "Broadcasted to", and added column "Viewable by" to tell the story. Hope I got it right, whew!